test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,53 @@
+require "test_helper"
+
+describe RecipesController do
+  describe 'new' do
+    it 'responds with success' do
+      get root_path
+
+      must_respond_with :success
+    end
+
+  end
+
+  describe 'index' do
+
+    it 'responds with success with valid data' do
+      VCR.use_cassette("recipes") do
+        get recipes_path, params: {query: "cheese"}
+
+        must_respond_with :success
+      end
+    end
+
+
+    it 'raises an error with invalid data' do
+
+        VCR.use_cassette("recipes") do
+        proc {
+          get recipes_path }.must_raise
```

Comment:
I like that you've identified this as a test case. I'm not sure that I agree with your decision about how to handle this situation - in general, the controller should catch exceptions and pass them on to the user as polite error messages. However, that doesn't invalidate the need to test this case.